doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-endpoints-ref.adoc

```diff
@@ -0,0 +1,4 @@
+[id='security-management-rest-api-endpoints-ref_{context}']
+= Supported Security Management REST API endpoints
+
+The Security Management REST API provides endpoints for managing groups, roles, users and permissions in {CENTRAL}. It includes the security and permission management tasks that an administrator can also perform using the *Security Management* page in {CENTRAL}.
```

Comment:
My assumption is that the security management REST API is part of the Knowledge Store REST API, based on where you have it in the doc and what the base URL is, etc. If that's the case, I would just make that explicit in the intro so people know that the way to make the API calls, authentication, etc. is the same as the Knowledge Store API, which is documented previously.

Something as simple as "The Security Management REST API **is part of the Knowledge Store REST API and** provides ... "

If technically this API isn't part of the Knowledge Store API, then we might want to broaden the parent chapter for this section to be something like "Business Central REST API ...", and likewise in the seciton how to use it, and then keep the division of Knowledge Store, Security Management, etc. as you have it. But check with and SME first, just to verify.

Reply:
@sterobin thanks for raising. 
As confirmed with @RishiRajAnand, we need to create a dedicated section for Security Management API similar to Knowledge Store. Therefore, I'll repeat the review cycle with Rishiraj and you for the dedicated Security Management API section.